fluent/ExtrinsicRune.ts

```diff
@@ -59,13 +59,30 @@ export class ExtrinsicRune<out C extends Chain, out U> extends PatternRune<Chain
       .fn(concat)
       .call(extrinsic, extrinsic.access("length").map((n) => $.u32.encode(n)))
       .map(hex.encodePrefixed)
-    const data = this.chain.connection
+    const info = this.chain.connection
       .call("state_call", "TransactionPaymentApi_query_info", arg)
       .map(hex.decode)
     return this.chain.metadata
-      .access("paths", "sp_weights::weight_v2::Weight")
-      .map(($c) => $.field("weight", $c))
+      .access("types", "DispatchInfo")
```

Comment:
These `types` are pretty unstable; you should use the corresponding `paths` codec instead.